tap/extensions/amqp/spec091.go

```diff
@@ -2854,7 +2856,8 @@ func (r *AmqpReader) parseMethodFrame(channel uint16, size uint32) (f frame, err
 			mf.Method = method
 
 		default:
-			return nil, fmt.Errorf("Bad method frame, unknown method %d for class %d", mf.MethodId, mf.ClassId)
+			// log.Printf("Bad method frame, unknown method %d for class %d", mf.MethodId, mf.ClassId)
```

Comment:
Either remove comment or print in DEBUG or TRACE level.
Same comment for the rest of the changes in this file.

Reply:
@nimrod-up9 removed them with https://github.com/up9inc/mizu/commit/1a6d32e6614e4587883b27dda599f011054b8605